tests/model/test_node.py

```diff
@@ -688,3 +688,274 @@ def test_multivariate_batched_distribution(Dist) -> None:
         dist.value,
         tfd.MultivariateNormalFullCovariance(loc_vec, scale_m).log_prob(x_vec),
     )
+
+
+class TestDistGetitem:
+    def test_anonymous_values_positional(self):
+        x = Dist(tfd.Normal, 0.0, 1.0)
+
+        assert x[0] is x.inputs[0]
+        assert x[1] is x.inputs[1]
+
+    def test_vars_named_kw(self):
+        loc = Var(0.0, name="loc")
+        scale = Var(1.0, name="scale")
+        x = Dist(tfd.Normal, loc=loc, scale=scale)
+
+        with pytest.raises(IndexError):
+            x[0]
+
+        with pytest.raises(IndexError):
+            x[1]
+
+        assert x["loc"] is loc
+        assert x["scale"] is scale
+
+    def test_vars_named_positional(self):
+        loc = Var(0.0, name="loc")
+        scale = Var(1.0, name="scale")
+        x = Dist(tfd.Normal, loc, scale)
+
+        assert x[0] is loc
+        assert x[1] is scale
+
+        with pytest.raises(KeyError):
+            x["loc"]
+
+        with pytest.raises(KeyError):
+            x["scale"]
+
+    def test_vars_unnamed(self):
```

Comment:
Do you think more descriptive test names to clearly stating intend could help making sure that we in the future know what should be tested? Could also be a comment.
For example here:  `test_vars_positional_access_to_named_args_raises_error`

Reply:
Oh, you are of course absolutely right. The test names were not very descriptive. I improved that now. I also removed some tests in `test_var.py` that have become redundant.